.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,61 @@
+name: Bug Report
+description: Something isn't working as expected
+labels: [bug]
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for contributing to Swift-DocC-Render! 
+      
+      Before you submit your issue, please complete each text area below with the relevant details for your bug, and complete the steps in the checklist
+- type: textarea
+  attributes:
+    label: Description
+    description: |
+      A short description of the incorrect behavior. 
+
+      If you think this issue has been recently introduced and did not occur in an earlier version, please note that. If possible, include the last version that the behavior was correct in addition to your current version.
+  validations:
+    required: true
+- type: checkboxes
+  attributes:
+    label: Checklist
+    options:
+    - label: If possible, I've reproduced the issue using the `main` branch of this package.
+      required: false
+    - label: This issue hasn't been addressed in an [existing GitHub issue](https://github.com/apple/swift-docc-render/issues).
+      required: true
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: Describe what you expected to happen.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Actual behavior
+    description: Describe or copy/paste the behavior you observe.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: |
+      Explanation of how to reproduce the incorrect behavior. 
+      
+      This could include a reduced version of your documentation bundle, or a link to the documentation (or code) that is exhibiting the issue.
+    placeholder: |
+      1. In this browser version...
+
+      2. With this viewport size...
+      
+      3. Run '...'
```

Comment:
Not sure what this would be referring to, since we don't run commands in Swift-DocC-Render. Is something like "Perform action…" more appropriate?

Reply:
I was thinking of a step like:

3. Run `VUE_APP_DEV_SERVER_PROXY=[path to docc] npm run serve`

Reply:
I expect that most issues will be filed after noticing a bug in some hosted version of a DocC Render website somewhere, rather than running things locally, so I don't think serving DocC Render locally should be part of the template.

Reply:
Makes sense! deleted d4ec0eeeda23beee35f7d5df4a662e79450e7d26